.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,41 @@
+name: Generate SBOM with Kubernetes BOM
+
+on:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
+
+jobs:
+  sbom:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    env:
+      OUTPUT: kube-state-metrics-${{ github.ref_name }}.spdx
+      TAG: ${{ github.ref_name }}
```

Comment:
This wouldn't work with event `release`, but should be resolved with 7fc6c5dd2e6596d6534587497c2821c090bf9555

See https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release